backend/tests/api/test_config.py

```diff
@@ -198,32 +322,69 @@ async def test_update_nonexistant_configuration(backend_client: AsyncClient):
         "license_server_type": "servertype100",
         "grace_time": "10000",
     }
+
+    inject_security_header("owner1", "license-manager:config:write")
     resp = await backend_client.put("/lm/api/v1/config/100000", json=data)
     assert resp.status_code == 200
 
 
 @mark.asyncio
 @database.transaction(force_rollback=True)
-async def test_delete_configuration(backend_client: AsyncClient, one_configuration_row, insert_objects):
+async def test_delete_configuration__success(
+    backend_client: AsyncClient,
+    one_configuration_row,
+    insert_objects,
+    inject_security_header,
+):
     """
     Test deleting a configuration row.
     """
 
     await insert_objects(one_configuration_row, table_schemas.config_table)
+
+    inject_security_header("owner1", "license-manager:config:write")
     resp = await backend_client.delete("/lm/api/v1/config/100")
     assert resp.status_code == 200
     assert resp.json()["message"] == "Deleted 100 from the configuration table."
 
 
+@mark.asyncio
+@database.transaction(force_rollback=True)
+async def test_delete_configuration__success(
```

Comment:
The name of this test is the same as the test above, it should be
```suggestion
async def test_delete_configuration__fail_on_bad_permission(
```

Reply:
Oooh, good catch! There was a lot of copy-pasta being thrown around! Not surprised some of it stuck to the wall.

Reply:
Fixed in latest commit.